examples/problems/1100_callback.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>PCA computation in gene space.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/moscot_notebooks/pull/70/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>